### ISSUE TITLE
Playwright issue for Mermaid support is disruptive

### ISSUE THREAD

**Issue by huijing** [interledger/docs-styleguide#22]
In order to add support for Mermaid diagrams in markdown, we made use of https://github.com/remcohaszing/remark-mermaidjs, which has a Playwright dependency under the hood. As it turns out, there is a very known issue around Playwright and browser versioning: https://gkushang.medium.com/playwright-failed-to-launch-browsers-how-to-solve-8b01d03fe5b9

And everyone seems to run into this issue regularly, but no one has a tried and tested way of ensuring it doesn't recur. Many solutions target Docker since that seems to be the predominant use-case but our static hosting solutions of Netlify and GitHub pages limit our configuration options.

For context, see https://github.com/interledger/docs-styleguide/issues/7 and https://github.com/interledger/rafiki/pull/2026

TLDR, this is not ideal, we are not even using Playwright directly. The logical course of action is to see if we can NOT use the plugin and roll our own Mermaid component directly using the library. The implication is that this will be a client-side solution, therefore we will also assess the performance implication of doing so.
